proto/stats.proto

```diff
@@ -13,3 +14,15 @@ message ContainerInfoReply {
 	optional string info = 1;
 	optional string error = 2;
 }
+
+message MetricsRequest {
```

Comment:
is it possible that we can pass in the name of the metrics we want? maybe we only care about cpu or io?